src/integrationTests/communities.test.ts

```diff
@@ -0,0 +1,218 @@
+import { Crypto } from '@peculiar/webcrypto';
+import {
+  createCommunity,
+  joinCommunity,
+  getCommunityOwnerData,
+  tryToJoinOfflineRegistrar,
+  assertReceivedCertificates,
+  assertReceivedChannelsAndSubscribe,assertReceivedMessages, launchCommunitiesOnStartupSaga, sendMessage
```

Comment:
typo. missing ' '

Reply:
I think we should configure linter the same way it is configured in ZbayLite and run --fix across the code